internal/developmentmembership/trackingsubjectset.go

```diff
@@ -84,24 +83,28 @@ func (tss *TrackingSubjectSet) Add(subjectsAndResources ...FoundSubject) error {
 	return nil
 }
 
-func keyFor(fs FoundSubject) string {
-	return fmt.Sprintf("%s#%s", fs.subject.Namespace, fs.subject.Relation)
+func typeKey(ns, rel string) string { return ns + "#" + rel }
```

Comment:
we repeat this a lot now... maybe move into the `tuple` package and make public and unit tested?